runtime/oti/VMHelpers.hpp

```diff
@@ -2115,17 +2119,16 @@ class VM_VMHelpers
 	static VMINLINE bool
 	tryWinningConcurrentGCScan(J9VMContinuation *continuation, bool isGlobalGC)
 	{
-		uintptr_t complementGCConcurrentState = J9_GC_CONTINUATION_STATE_CONCURRENT_SCAN_NONE;
+		bool ret = false;
+		volatile uintptr_t *localAddr = &continuation->state;
+		uintptr_t complementGCConcurrentState = *localAddr;
```

Comment:
this variable does not have the same meaning as before (when we really extracted complement GC concurrent state)
it could just be simple called continuationState